### PR TITLE
refactor (html5): Provide list of available locales through settings.yml (and Graphql)

### PR DESCRIPTION
**Pull request by gustavotrott**
TL;DR - basically set all available langs in `settings.yml` instead of discovering it from file system (finding `.json` files in locales dir).

Motivation: Stop depending on Meteor for providing available locales.

----

<table>
<tr>
<td>

image

</td>
<td>

image

</td>
</tr>
</table>

The current method for listing available locales in Meteor is through the endpoint `/html5client/locale-list` and the data is also stored in the MongoDB `captions` collection.

Meteor identifies available locales by locating .json files within the directory `/usr/share/meteor/bundle/programs/web.browser/app/locales`:

image

This method lacks flexibility as administrators cannot easily select which languages to provide, and it depends on a predefined file path.

To improve flexibility and ease of configuration, this PR proposes eliminating the dependency on the file system for locale discovery. 
Instead, we will use a new configuration entry `public.captions.locales` in the `settings.yml` file, which will list languages and their corresponding names directly (_e.g., en: English_). This list will then be accessible through the GraphQL API rather than the existing Meteor API, streamlining the process and making it configurable directly from the settings file.

Graphql query:
```gql
query {
  meeting_clientSettings {
    locales: clientSettingsJson(path: "$.public.captions.locales")
  }
}
```

```json
{
  "data": {
    "meeting_clientSettings": [
      {
        "locales": [
          {
            "name": "Afrikaans",
            "locale": "af"
          },
          {
            "name": "العربية",
            "locale": "ar"
          },
          {
            "name": "Azərbaycan dili",
            "locale": "az"
          },
          {
            "name": "Български",
            "locale": "bg-BG"
          },
          {
            "name": "বাংলা",
            "locale": "bn"
          },
          {
            "name": "Català",
            "locale": "ca"
          },
          {
            "name": "Čeština",
            "locale": "cs-CZ"
          },
          {
            "name": "Dansk",
            "locale": "da"
          },
          {
            "name": "Deutsch",
            "locale": "de"
          },
          {
            "name": "ދިވެހި",
            "locale": "dv"
          },
          {
            "name": "Ελληνικά",
            "locale": "el-GR"
          },
          {
            "name": "English",
            "locale": "en"
          },
          {
            "name": "Esperanto",
            "locale": "eo"
          },
          {
            "name": "Español",
            "locale": "es"
          },
          {
            "name": "Español (Latinoamérica)",
            "locale": "es-419"
          },
          {
            "name": "Español (España)",
            "locale": "es-ES"
          },
          {
            "name": "Español (México)",
            "locale": "es-MX"
          },
          {
            "name": "eesti keel",
            "locale": "et"
          },
          {
            "name": "Euskara",
            "locale": "eu"
          },
          {
            "name": "فارسی",
            "locale": "fa-IR"
          },
          {
            "name": "Suomi",
            "locale": "fi"
          },
          {
            "name": "Français",
            "locale": "fr"
          },
          {
            "name": "Galego",
            "locale": "gl"
          },
          {
            "name": "עברית‏",
            "locale": "he"
          },
          {
            "name": "हिन्दी",
            "locale": "hi-IN"
          },
          {
            "name": "Hrvatski",
            "locale": "hr"
          },
          {
            "name": "Magyar",
            "locale": "hu-HU"
          },
          {
            "name": "Հայերեն",
            "locale": "hy"
          },
          {
            "name": "Bahasa Indonesia",
            "locale": "id"
          },
          {
            "name": "Italiano",
            "locale": "it-IT"
          },
          {
            "name": "日本語",
            "locale": "ja"
          },
          {
            "name": "ქართული",
            "locale": "ka"
          },
          {
            "name": "ភាសាខ្មែរ",
            "locale": "km"
          },
          {
            "name": "ಕನ್ನಡ",
            "locale": "kn"
          },
          {
            "name": "한국어 (韩国)",
            "locale": "ko-KR"
          },
          {
            "name": "ລາວ",
            "locale": "lo-LA"
          },
          {
            "name": "Lietuvių",
            "locale": "lt-LT"
          },
          {
            "name": "Latviešu",
            "locale": "lv"
          },
          {
            "name": "മലയാളം",
            "locale": "ml"
          },
          {
            "name": "Монгол",
            "locale": "mn-MN"
          },
          {
            "name": "Norsk (bokmål)",
            "locale": "nb-NO"
          },
          {
            "name": "Nederlands",
            "locale": "nl"
          },
          {
            "name": "Occitan",
            "locale": "oc"
          },
          {
            "name": "Polski",
            "locale": "pl-PL"
          },
          {
            "name": "Português",
            "locale": "pt"
          },
          {
            "name": "Português (Brasil)",
            "locale": "pt-BR"
          },
          {
            "name": "Română",
            "locale": "ro-RO"
          },
          {
            "name": "Русский",
            "locale": "ru"
          },
          {
            "name": "Slovenčina (Slovakia)",
            "locale": "sk-SK"
          },
          {
            "name": "Slovenščina",
            "locale": "sl"
          },
          {
            "name": "Српски",
            "locale": "sr"
          },
          {
            "name": "Svenska",
            "locale": "sv-SE"
          },
          {
            "name": "தமிழ்",
            "locale": "ta"
          },
          {
            "name": "తెలుగు",
            "locale": "te"
          },
          {
            "name": "ภาษาไทย",
            "locale": "th"
          },
          {
            "name": "Türkçe",
            "locale": "tr-TR"
          },
          {
            "name": "Українська",
            "locale": "uk-UA"
          },
          {
            "name": "Tiếng Việt",
            "locale": "vi-VN"
          },
          {
            "name": "中文（中国",
            "locale": "zh-CN"
          },
          {
            "name": "中文（台灣",
            "locale": "zh-TW"
          }
        ]
      }
    ]
  }
}
```

### CODE REVIEW

**Review by antobinary — APPROVED** [review]
I understand the idea and it sounds good.

**Review by Tainan404 — APPROVED** [review]
